src/renderers/dom/shared/__tests__/ReactDOMComponent-test.js

```diff
@@ -1108,6 +1108,27 @@ describe('ReactDOMComponent', () => {
       expectDev(console.log.calls.argsFor(0)[0]).toContain('onError called');
       expectDev(console.log.calls.argsFor(1)[0]).toContain('onLoad called');
     });
+
+    it('should support onLoad and onError events on SVG element', () => {
+      var container = document.createElement('div');
+
+      spyOn(console, 'log');
+
+      ReactDOM.render(
+        <svg
+            onError={() => console.log('onError called')}
+            onLoad={() => console.log('onLoad called')}
+        />,
+        container
+      );
+
+      container.getElementsByTagName('svg')[0].dispatchEvent(new Event('error'));
+      container.getElementsByTagName('svg')[0].dispatchEvent(new Event('load'));
+
+      expectDev(console.log.calls.count()).toBe(2);
+      expectDev(console.log.calls.argsFor(0)[0]).toContain('onError called');
+      expectDev(console.log.calls.argsFor(1)[0]).toContain('onLoad called');
+    });
```

Comment:
Could you use [`jest.fn()`](http://facebook.github.io/jest/docs/en/jest-object.html#jestfnimplementation). You could run an assertion like:

```
const onError = jest.fn()
const onLoad = jest.fn()

ReactDOM.render(<svg onError={onError} onLoad={onLoad} />, container)

expect(onError).toHaveBeenCalled()
expect(onLoad).toHaveBeenCalled()
```